ac-dcd.el

```diff
@@ -596,7 +596,7 @@ dcd-client outputs candidates that begin with \"this\" when completing struct co
         (ac-dcd-goto-def-push-marker)
         (unless (string=  file "stdin") ; the declaration is in the current file
           (find-file file))
-        (goto-char (byte-to-position (string-to-number offset)))))))
+        (goto-char (byte-to-position (1+ (string-to-number offset))))))))
```

Comment:
I'm sure this fixes an issue for you, but I'm wondering how this is the fix. Given your description, it should return 0 if it's negative, but I don't know why you'd change the existing positive values.

Reply:
The path to error is

(offset): "0"
(string-to-number "0"): 0
(byte-to-position (string-to-number offset)): nil
(goto-char nil): Errors as: (wrong-type-argument integer-or-marker-p nil)

Reply:
No. I don't have any issue at all right now. Which makes it all the harder for me to understand to me why 1 should be added in all cases and not just when it's 0.